lib/propolis/src/hw/virtio/viona.rs

```diff
@@ -475,6 +504,16 @@ impl PciVirtio for PciVirtioViona {
     fn pci_state(&self) -> &pci::DeviceState {
         &self.pci_state
     }
+    fn notify_port_update(&self, port: Option<u16>) {
```

Comment:
It might be worth a comment explaining the difference between `notify_port_update` and `set_notify_iop`, which this calls if the device is running? It could be helpful if we explained when this should be called, and when it's okay to call `set_notify_iop` unconditionally?